modules/addons/openprovider/Controllers/System/SynchroniseController.php

```diff
@@ -198,7 +198,7 @@ public function processNotUpdatedDomains()
                 // Update the registrar with to openprovider.
                 $domain =  $scheduled_domain_transfer->tbldomain;
                 $domain->registrar = 'openprovider';
-                $domain->expirydate = $openprovider_domain['expirationDate'];
+                $domain->expirydate = $openprovider_domain['renewalDate']; // Use renewal date as expiry date.
```

Comment:
@Uditha-05 Few concerns, can you please check and feedback:

- expirydate is all lowercase and not camel case, is that fine?
- Also, I didn't find the conditional check of module configuration and we always are updating renewalDate, is that correct?:
![image](https://github.com/user-attachments/assets/f5318dfc-b918-4cae-bbde-8be3beac53ae)

Thank you.

Reply:
1. Yes. The variable names are in lowercase. (came from the previous version)

2. Several reasons behind that. The configuration file is not used in the **add-on** module. But it was changed to avoid keeping "expirationDate" further and in the syncing time it should be "renewalDate". 
Also this function in "modules/addons/openprovider/Controllers/System/SynchroniseController.php" is not used anymore. ( `OP sync cron` is deprecated and then this controller function will not be used further. [Link for README](https://github.com/openprovider/Openprovider-WHMCS-domains/blob/master/README.md#deprecated-configure-the-openprovider-cron-sync))